lld/test/ELF/systemz-shared.s

```diff
@@ -0,0 +1,86 @@
+// REQUIRES: systemz
```

Comment:
This test, copied from `shared.s`, is probably not very useful. Everything tested is generic and tested in many other places.

Reply:
Removed test.